src/main.cpp

```
@@ -149,21 +160,73 @@ void initializeSystem() {
+        
+        // Mostrar en display si está disponible
+        #ifndef STANDALONE_DISPLAY
+        HUD::showError();
+        #endif
+        
+        Serial.println("[CRITICAL ERROR] Max retries - stopping watchdog feeds");
+        Serial.flush();
+        
+        // Detener watchdog feeds - sistema se reseteará en 30 segundos
+        while (true) {
+            delay(1000);
```

Clang (14.0.6)

[warning] 1000 is a magic number; consider replacing it with a named constant

(cppcoreguidelines-avoid-magic-numbers,readability-magic-numbers)